package.json

```diff
@@ -11,7 +11,7 @@
 		"test": "npm run tests-only && npm run test:harmony",
 		"posttest": "npx aud --production",
 		"tests-only": "nyc tape test",
-		"test:harmony": "nyc node --es-staging --harmony test",
+		"test:harmony": "nyc node --harmony test",
```

Comment:
this is required in certain node versions to expose arrow functions - v0.12 i believe?

Reply:
> this is required in certain node versions to expose arrow functions - v0.12 i believe?

Gotcha, I'll re-add. It was messing up my tests locally, but I can still run them with `tests-only`